WireAPI/Sources/WireAPI/Authentication/CookieStorage.swift

```diff
@@ -17,6 +17,7 @@
 //
 
 import Foundation
+import WireCrypto
```

Comment:
why WireAPI needs WireCrypto?

Reply:
It uses `static AES256Crypto.encryptAllAtOnceWithPrefixedIV(plaintext:key:)` which was in `WireFoundation` but moves to `WireCrypto` in this PR.

Reply:
@netbe I'm resolving this comment. Feel free to re-open if you would like to discuss further.